svd-parser/src/expand.rs

```diff
@@ -8,98 +8,152 @@ use svd_rs::{
     Device, EnumeratedValues, Field, Peripheral, Register, RegisterCluster, RegisterProperties,
 };
 
+#[derive(Clone, Debug, PartialEq, Hash, Eq)]
+pub struct RPath {
```

Comment:
I think it's ok to have these have a longer name

```suggestion
pub struct RegisterPath {
```
keeping it is fine too though, it may just be unclear for someone not familiar with how this works and looking here first time

Reply:
Technically this is not path of register but path to register/cluster/fields. Like a directory.